params.py

```diff
@@ -86,6 +86,8 @@ def load_params(args):
                 mysuite.reportCoverage = mysuite.reportCoverage or value
             elif opt == "emailTo":
                 mysuite.emailTo = value.split(",")
+            elif opt == "ftools":
+                mysuite.ftools = value
```

Comment:
Is this the right syntax? not sure if this creates an array, but it seems to be used that way for `extra_tools`